doc/source/getting_started/installation.rst

```diff
@@ -1,92 +1,69 @@
-.. _installation:
+.. _ref_installation:
 
 ============
 Installation
 ============
 
-The ``ansys-fluent-core`` package supports Python 3.9 through
-Python 3.12 on Windows and Linux.
 
-.. note::
-   This page provides information for quickly installing and launching
-   the ``ansys-fluent-core`` package. Additional PyFluent packages, which
-   are described later on this page, can also be installed. For
-   information on installing all PyFluent packages in a virtual environment,
-   see :ref:`faqs_install` in :ref:`faqs`.
+PyFluent Installation
+---------------------
 
-Install the package
--------------------
-Install the latest ``ansys-fluent-core`` package from
-`PyPi <https://pypi.org/project/ansys-fluent-core/>`_ with this code:
+PyFluent supports Python 3.9 through Python 3.12 on Windows, Mac OS and Linux.
+
+PyFluent can be installed, along with all its optional dependencies, using:
 
 .. code::
 
    pip install ansys-fluent-core
 
-If you plan on doing local *development* of PyFluent with Git, install the
-latest ``ansys-fluent-core`` package with this code:
 
+Development Installation
+------------------------
+
+The PyFluent source repository is available on GitHub. You can clone the repository using:
 .. code:: console
 
    git clone https://github.com/ansys/pyfluent.git
```

Comment:
Shall we do `cd pyfluent` immediately after this step? User will need to move to the pyfluent directory before executing `python codegen/allapigen.py`.

Reply:
I think the idea in this part of the code is not to provide the full script, this part is just for the "clone repository command"

The codegen section is down below, that is where we need the `cd pyfluent`, or need to let the user know that they need to move into the local PyFluent repository folder before executing codegen

Reply:
Also, I think I preferred providing the full script in one block as we did before:
```
git clone https://github.com/ansys/pyfluent.git
cd pyfluent
pip install pip -U
pip install -e .
python codegen/allapigen.py  # Generates the API files
```
And then we could explain the commands afterwards?

Reply:
One thing to keep in mind that user will tend to copy and execute the code verbatim from the website, especially when there is a copy button next to the code.

Reply:
Yes, 'cd pyfluent` step should be mentioned here before  executing `python codegen/allapigen.py`

Reply:
I'll mostly revert this section

Reply:
It's useful to have the copyable script in a single block. 

We also want each step to be fully explained. I will move all those explanations up into a single block.

I will remove the comment # Generates the API files in favour of the fuller, separate explanation.

Reply:
TODO create an issue for the outstanding aspects here.

Reply:
**This section has been fully updated.**

Reply:
marking resolved but feel free to reopen